shell/ci/testing/delibird.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+# Runs the delibird log/tracing/session-recording uploader if
+# configured to do so in the box configuration.
+#
+# Automatically handles installation of the delibird log uploader if it
+# is not already installed.
+set -euo pipefail
+
+# shellcheck source=../../lib/bootstrap.sh
+source "$DIR/../../lib/bootstrap.sh"
+# shellcheck source=../../lib/logging.sh
+source "$DIR/../../lib/logging.sh"
+# shellcheck source=../../lib/github.sh
+source "$DIR/../../lib/github.sh"
+
+# DELIBIRD_ENABLED denotes if the delibird log uploader should be
+# enabled or not. If the value is "true", then the delibird log uploader
+# will be enabled. If the value is "false", then the delibird log
+# uploader will be disabled.
+DELIBIRD_ENABLED=$(get_box_field ".delibird.enabled")
```

Comment:
`get_box_field` returns an empty string if it's not present ([see test](https://github.com/getoutreach/devbase/pull/636/commits/1cec8ca8756698237afe5b9e3c14fb903148eabd)). The default is that this would _not_ be set, this being set to `false` by default (which we want!).

I will have a PR to update our box configuration to enable it for us, once this merges!